nextflow.config

```diff
@@ -46,7 +46,7 @@ profiles {
     params.goldstandard_id = "syn58888786"
     params.private_folders = "predictions"
     params.challenge_container = "ghcr.io/sage-bionetworks-challenges/pegs-evaluation:latest"
-    params.send_email = false
```

Comment:
These are a confusing set of parameters - Do we have any control to update this?

I would assume that we would be setting:
`send_email = true`

at a base level that determines Yes/No we are sending at an email at all. Then come the flags to modify the content that would be included in the email such as:

`include_score_in_email` True/False

Reply:
I see from the docs that `send_email` defaults to true. Since we are modifying the flags around it I would keep `send_email = true` in this config so it isn't misinterpreted that `email_with_score` is controlling it:

`1. `send_email` (optional): If `true`, sends an e-mail to the submitter on the status of their submission. Default is `true`.`

Either way I understand now how this is set up.

Reply:
I think this is a good point- There's a little bit of cleanup I can do with the way e-mail templates are chosen (with/without score). One of them being the wording and the other being why one is handled with a boolean and the other with a string. I'll make a quick tech debt ticket for this.